plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -285,22 +285,20 @@ export default class ChatComposer extends Component {
   }
 
   @action
-  onTextareaFocusOut(event) {
+  onTextareaFocusOut() {
     this.isFocused = false;
-    enableBodyScroll(event.target);
```

Comment:
removing enable/disable here is unrelated, but we where doing twice the work given this optimization is already done in the composer service when we focus.